src/cev/_embedding_widget.py

```diff
@@ -65,7 +65,7 @@ def _validate_labels(self, proposal: object):
         # convert to category if not already
         return (
             proposal.value
-            if proposal.value == "category"
+            if proposal.value.dtype.name == "category"
```

Comment:
hmm, shouldn't this be

```suggestino
            if proposal.value.dtype == "category"
```

Reply:
or maybe even [this](https://pandas.pydata.org/docs/reference/api/pandas.api.types.is_categorical_dtype.html) is better:

```suggestion
            if not pd.api.types.is_categorical_dtype(proposal.value)
```